components/raftstore/src/store/msg.rs

```diff
@@ -416,6 +417,33 @@ impl<S: Snapshot> RaftCommand<S> {
     }
 }
 
+pub struct TracedMsg<M> {
+    pub trace_scope: Scope,
+    pub msg: M,
+}
+
+impl<M> TracedMsg<M> {
+    pub fn new(message: M, trace_event_name: &'static str) -> Self {
+        Self {
+            trace_scope: child_scope(trace_event_name),
```

Comment:
How about `Scope::child()` to be aligned with `Scope::default()`?

Reply:
My idea is `Scope::root()`, `Scope::child()`, `Scope::empty()`.